airgun/views/common.py

```diff
@@ -115,6 +115,11 @@ class listremove(SatSecondaryTab):
     ROOT = ParametrizedLocator(
         './/nav[@class="ng-scope" or not(@*)]/following-sibling::div')
 
+    TAB_LOCATOR = ParametrizedLocator(
```

Comment:
what was a reason for this change?

Reply:
since it wasn't overriding TAB_LOCATOR before, both primary and secondary tabs could be caught by this locator. It's just we were lucky every single automated page had different labels for primary and secondary tabs; now i faced the page where we have both primary and secondary 'Details' tab, this way the issue was uncovered :)

Reply:
so it will not affect existing tests?

Reply:
The only secondary tabs we used till now are common 'Add' and 'List/Remove' tabs for resources lists. robottelo's PR does have test results for them